profile/README.md

```diff
@@ -7,3 +7,10 @@ The application was created so that Ukrainians could interact with the state in
 
 # Get started
 Open Source, or Open Code, is software with code that can be seen by anyone. Open Source shows the quality of programming as well as the logic of application's operation. By making Diia's code publicly available, we gave the opportunity to review, edit, or even modify it by any techie.
+
+[![SWUbanner]][SWUdocs]
+
+[SWUbanner]:
+https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner-direct-team.svg
+[SWUdocs]:
+https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md
```

Comment:
Perhaps we should use the link to the SWU site:
```suggestion
https://stand-with-ukraine.pp.ua/
```

This is according to the author's doc: https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/AddBanner.md#for-a-project-developed-by-the-team

Reply:
Most of the recommendations link GitHub because it's not likely to be blocked for people.